### PR TITLE
Translation corrections

### PR DESCRIPTION
**Pull request by Tymaeus**
Various translation corrections.
Given the number of translation errors in this script alone, it is my belief that the whole route needs a second glance over for translation accuracy corrections.

### CODE REVIEW

File: Update/kage_001.txt [hunk 1]
```diff
@@ -5614,7 +5614,7 @@ void main()
 			NULL, "\"Ahahaha, that's right~.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 36, "ps3/s08/36/320100030", 256, TRUE);
 	OutputLine(NULL, "エンジェルモートは前の町にもあったけど、こんなに美味しいケーキはなかったかなぁ」",
-			NULL, " Angel Mort has a shop in my old town. I wonder if their cakes are this delicious too?\"", Line_Normal);
+			NULL, " Angel Mort has a shop in my old town. but I don't think they ever had cakes this delicious.\"", Line_Normal);
```
Comment (drojf) [inline Update/kage_001.txt:5617]:
```suggestion
			NULL, " Angel Mort has a shop in my old town, but I don't think they ever had cakes this delicious.\"", Line_Normal);
```

File: Update/kage_001.txt [hunk 2]
```diff
@@ -5300,7 +5300,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 37, "ps3/s08/37/320300042", 256, TRUE);
 	OutputLine(NULL, "「決まってるでしょ、健全な男女交際のことよ！",
-			NULL, "\"You need to decide. It's necessary for healthy relationships!", Line_WaitForInput);
+			NULL, "\"It's obvious, isn't it? It's healthy act for any relationship!", Line_WaitForInput);
```
Comment (drojf) [inline Update/kage_001.txt:5303]:
```suggestion
			NULL, "\"It's obvious, isn't it? It's a healthy act for any relationship!", Line_WaitForInput);
```

File: Update/kage_001.txt [hunk 3]
```diff
@@ -5377,7 +5377,7 @@ void main()
 			NULL, "\"...Don't look to me for agreement.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 38, "ps3/s08/38/320400032", 256, TRUE);
 	OutputLine(NULL, "告白されたばかりの女の子にそんな質問するなんて、あんたの発想の起源は美少女ゲームか？」",
-			NULL, " I don't want to ask a question like that to a girl who just received a confession. Do you think this is the introduction to a dating sim?\"", Line_Normal);
+			NULL, " Where'd you get the idea to ask a girl who just received a confession something like that, a dating sim?\"", Line_Normal);
```
Comment (drojf) [inline Update/kage_001.txt:5380]: The sentence doesn't make sense to me (the ", a dating sim?" part  just seems appended to the sentence)

File: Update/kage_001.txt [hunk 4]
```diff
@@ -5676,7 +5676,7 @@ void main()
 //rふとつま先に軽い衝撃を感じて、顔を上げる。…見ると正面に座る珠子が、私をとがめるように険しい視線を向けていた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "ふとつま先に軽い衝撃を感じて、顔を上げる。…見ると正面に座る珠子が、私をとがめるように険しい視線を向けていた。",
-			NULL, "Sensing a bit of a reaction, I looked up. ...And I saw Tamako right in front of me, throwing a glare of condemnation in my direction.", Line_Normal);
+			NULL, "Suddenly, I felt something poking my toes and looked up. ...When I did, I saw Tamako throwing a glare of condemnation in my direction from across the table.", Line_Normal);
```
Comment (drojf) [inline Update/kage_001.txt:5679]: There's some grammar missing...I filled it in myself, but it doesn't flow well. Currently it sounds like the thing which is poking your toes is looking up.

```suggestion
			NULL, "Suddenly, I felt something poking my toes, so I looked up. ...When I did, I saw Tamako throwing a glare of condemnation in my direction from across the table.", Line_Normal);
```

**Review by drojf — CHANGES_REQUESTED** [review]
minor fixes and comments

I read your comment about a look-over being required, but thought I might as well point out what I saw anyway.

btw, normally I don't read the console arc repo commits, I just happened to do it just now

File: Update/kage_001.txt (hunk 3 above)
Comment (drojf) [reply Update/kage_001.txt:5380]: Actually I get it, never mind :S

File: Update/kage_001.txt [hunk 5]
```diff
@@ -5454,7 +5454,7 @@ void main()
 //r守先輩が持ってきたケーキを見て、珠子がため息交じりの歓声を上げる。…それはケーキ屋でも最上段にありそうな、カラフルな彩のストロベリーケーキだった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "守先輩が持ってきたケーキを見て、珠子がため息交じりの歓声を上げる。…それはケーキ屋でも最上段にありそうな、カラフルな彩のストロベリーケーキだった。",
-			NULL, "Looking at the cake Mamoru brought over, Tamako let out a sigh of relief and a cheer. ...It was a brightly colored strawberry cake, the kind you'd see on display in the front shelf at a bakery.", Line_Normal);
+			NULL, "Looking at the cake Mamoru brought over, Tamako let out a happy sigh. ...It was a brightly colored strawberry cake, the kind you'd see on display in the front shelf at a bakery.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/kage_001.txt:5457]:
```suggestion
			NULL, "Looking at the cake Mamoru brought over, Tamako let out a joyful sigh. ...It was a brightly colored strawberry cake, the kind you'd see on display in the front shelf at a bakery.", Line_Normal);
```

File: Update/kage_001.txt [hunk 6]
```diff
@@ -5568,13 +5568,13 @@ void main()
 //r珠子は何かを感づいたように、夏美はとにかく恐縮しきった顔で、守先輩に頭を下げる。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "珠子は何かを感づいたように、夏美はとにかく恐縮しきった顔で、守先輩に頭を下げる。",
-			NULL, "As Tamako noticed the look of frustration on Natsumi's face, she bowed to Mamoru.", Line_Normal);
+			NULL, "Tamako seemed to have realized what was up, while Natsumi bowed gratefully Mamoru.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/kage_001.txt:5571]:
```suggestion
			NULL, "Tamako seemed to have realized what was up, while Natsumi gratefully bowed to Mamoru.", Line_Normal);
```

File: Update/kage_001.txt [hunk 7]
```diff
@@ -5568,13 +5568,13 @@ void main()
...
 	ClearMessage();
 
 //rそして、カウンターに戻っていったその背中を見送ってから、私たちはフォークを持ってケーキを掬い取り、——ぱくり、と食べた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "そして、カウンターに戻っていったその背中を見送ってから、私たちはフォークを持ってケーキを掬い取り、——ぱくり、と食べた。",
-			NULL, "Then he went back behind the counter, so the three of us grabbed our forks, opened our mouth wide... and scooped up some cake.", Line_Normal);
+			NULL, "Then, after we watched him go back behind the counter, the three of us grabbed our forks, opened our mouth wide... and scooped up some cake.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/kage_001.txt:5577]:
```suggestion
			NULL, "Then, after we watched him go back behind the counter, the three of us grabbed our forks, opened our mouths wide... and scooped up some cake.", Line_Normal);
```

File: Update/kage_001.txt [hunk 8]
```diff
@@ -5742,10 +5742,10 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 38, "ps3/s08/38/320400042", 256, TRUE);
 	OutputLine(NULL, "『千紗登のマイペースはいいところもあるけどさ、",
-			NULL, "\"Chisato, there's a time and place for the way you do things,", Line_WaitForInput);
+			NULL, "\"Chisato, the way you keep things marching to your own tune is charming and all,", Line_WaitForInput);
 	ModPlayVoiceLS(4, 38, "ps3/s08/38/320400043", 256, TRUE);
 	OutputLine(NULL, "…そのせいで相手の気持ちに鈍感なのは全然よくない』",
-			NULL, " ...but your complete lack of consideration for the feelings of others is problematic.\"", Line_Normal);
+			NULL, " ...but as a result, it's also left you unwittingly inconsiderate of the way others feel. That's not admirable at all.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/kage_001.txt:5748]:
```suggestion
			NULL, " ...but as a result, it's also left you unwittingly inconsiderate of the feelings of others. That's not admirable at all.\"", Line_Normal);
```

File: Update/kage_001.txt [hunk 9]
```diff
@@ -5754,17 +5754,17 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 38, "ps3/s08/38/320400044", 256, TRUE);
 	OutputLine(NULL, "『あんたは意識してないかもしれないけど、自分以外のものを比較対象にして自分の幸せをアピールするやり方、",
-			NULL, "\"You may not realize it, but you find happiness by comparing yourself to others...", Line_WaitForInput);
+			NULL, "\"You may not realize it, but the way you compare things other than yourself and constantly talk about what makes you happy...", Line_WaitForInput);
 	ModPlayVoiceLS(4, 38, "ps3/s08/38/320400045", 256, TRUE);
 	OutputLine(NULL, "…聞く相手によっては、笑えない時もあるよ』",
-			NULL, " Depending on who you're talking to, there are times when you can't laugh.\"", Line_Normal);
+			NULL, " may not be so amusing to some people at times.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/kage_001.txt:5760]:
```suggestion
			NULL, " sometimes, the people you're dealing with may not find it so amusing.\"", Line_Normal);
```

**Review by DoctorDiablo — APPROVED** [review]

**Pull request merged**